lib/ui/card.tsx

```diff
@@ -2,14 +2,25 @@ import * as React from "react"
 
 import { cn } from "@/lib/utils"
 
+import ArrowRight from "@/icons/ArrowRight"
+import InfoCircleLine from "@/icons/InfoCircleLine"
+
+import { Icon } from "@/components/Utilities/Icon"
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/ui/tooltip"
+
 const Card = React.forwardRef<
   HTMLDivElement,
   React.HTMLAttributes<HTMLDivElement>
 >(({ className, ...props }, ref) => (
   <div
     ref={ref}
     className={cn(
-      "rounded-2xl border border-solid border-border bg-card text-card-foreground shadow-sm",
+      "min-w-60 rounded-2xl border border-solid border-border bg-card text-card-foreground",
```

Comment:
Not entirely sold we need a `min-w`, this should be a last resort I think. If you're doing this so things display better on the storybook example, we can easily create a decorator that wraps the card inside a container that has a min width and height.

Reply:
Done! Added decorators to stories :)